safe/messaging/item/abstract_list.py

```diff
@@ -65,6 +65,10 @@ def add(self, item):
         elif isinstance(item, QPyNullVariant):
             self.items.append(PlainText(
                 tr('Null (PyQt4.QtCore.QPyNullVariant) found from the data.')))
+        elif isinstance(item, tuple) or isinstance(item, list):
```

Comment:
Btw, there is another way to say this, it's `isinstance(item, (list, tuple))`. No need to change, just sharing :D

Reply:
arf, didn't know that. Thanks. Yes I keep it for next time ;-)